vmdb/lib/miq_automation_engine/service_models/miq_ae_service_miq_request.rb

```diff
@@ -28,5 +28,8 @@ def set_message(value)
       object_send(:update_attribute, :message, value)
     end
 
+    def description=(new_description)
+      object_send(:update_attribute, :description, new_description)
```

Comment:
It's weird that the setter would do the update.  Normally, an accessor just sets it and then a later call to something like `.save` would save it.  Can you just use the normal `expose` for this?  Is `save` exposed, or perhaps only exposed in a safe way?